index.html

```diff
@@ -38,8 +45,8 @@ <h2 class="section h2">인기 상품을<br>확인해보세요</h2>
             <div class="box">
                 <div class="text">
                     <span class="section hot-item">Hot item</span>
```

Comment:
❗️ 수정요청
아래 문구를 바꿔주신 것 처럼 디자인을 확인하시고 여기 문구도 변경해주세요!
```suggestion
                    <span class="section hot-item">Search</span>
```